decidim-elections/app/views/decidim/votings/census/admin/census/_export_codes.html.erb

```diff
@@ -0,0 +1,3 @@
+<div class="card-section">
+  Export the access codes nau!
```

Comment:
🥸

Reply:
That's taken care of in https://github.com/decidim/decidim/pull/7756 :)